src/app/legacy/psammead/psammead-social-embed/src/index.test.jsx

```diff
@@ -384,6 +402,16 @@ describe('AmpSocialEmbed', () => {
         />,
       );
 
+      const hasConsentBanner =
+        embed.oembed.provider_name === 'youtube' ||
+        embed.oembed.provider_name === 'facebook' ||
+        embed.oembed.provider_name === 'instagram';
+
+      if (hasConsentBanner) {
+        const button = screen.getByTestId('banner-button');
+        fireEvent.click(button);
+      }
+
```

Comment:
This check may be able to be removed? Can be a follow up change though if the tests pass.

Reply:
Yeah we can do a follow up, this doesn't button invoking doesn't fix the test so needs to be looked at anyway